acceptance/acceptance_test.go

```diff
@@ -143,7 +144,7 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 	for _, dir := range testDirs {
 		testName := strings.ReplaceAll(dir, "\\", "/")
 		t.Run(testName, func(t *testing.T) {
-			if !InprocessMode {
+			if !InprocessMode && !hasCustomServer(t, dir) {
```

Comment:
Why should having custom server disable parallel runs?

Reply:
Fixed. Originally I did that since I used `t.Setenv`. I modified the approach to simply set override the command environment variable instead.